datasets/lama/lama.py

```diff
@@ -0,0 +1,390 @@
+                    "masked_sentence": datasets.Value("string"),
+                    "negated": datasets.Value("string"),
+                }
+            )
+            return datasets.DatasetInfo(
+                description=_DESCRIPTION,
+                features=features,
+                supervised_keys=None,
+                homepage=_HOMEPAGE,
+                license=_LICENSE,
+                citation=_CITATION,
+            )
+        elif self.config.name == "squad":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "sub_label": datasets.Value("string"),
+                    "obj_label": datasets.Value("string"),
+                    "negated": datasets.Value("string"),
+                    "masked_sentence": datasets.Value("string"),
+                }
+            )
+            return datasets.DatasetInfo(
+                description=_DESCRIPTION,
+                features=features,
+                supervised_keys=None,
+                homepage=_HOMEPAGE,
+                license=_LICENSE,
+                citation=_CITATION,
+            )
+        elif self.config.name == "google_re":
+            features = datasets.Features(
+                {
+                    "pred": datasets.Value("string"),
+                    "sub": datasets.Value("string"),
+                    "obj": datasets.Value("string"),
+                    "evidences": datasets.Value("string"),
+                    "judgments": datasets.Value("string"),
+                    "sub_w": datasets.Value("string"),
+                    "sub_label": datasets.Value("string"),
+                    "sub_aliases": datasets.Value("string"),
+                    "obj_w": datasets.Value("string"),
+                    "obj_label": datasets.Value("string"),
+                    "obj_aliases": datasets.Value("string"),
+                    "uuid": datasets.Value("string"),
+                    "masked_sentence": datasets.Value("string"),
+                    "template": datasets.Value("string"),
+                    "template_negated": datasets.Value("string"),
+                }
+            )
+            return datasets.DatasetInfo(
+                description=_DESCRIPTION,
+                features=features,
+                supervised_keys=None,
+                homepage=_HOMEPAGE,
+                license=_LICENSE,
+                citation=_CITATION,
+            )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        my_urls = _URLs[self.config.name]
+        data_dir = dl_manager.download_and_extract(my_urls)
+        if self.config.name == "trex":
+            return [
+                datasets.SplitGenerator(
+                    name=datasets.Split.TRAIN,
+                    gen_kwargs={
+                        "filepath": [
+                            os.path.join(data_dir, *f.split("/"))
+                            for f in [
```

Comment:
maybe you can use `glob` here instead ?

This would also allow to have only 1 or 2 of those files in the dummy_data.zip instead of all of them

Reply:
done